src/components/ChatList/GroupChatChannelItem.tsx

```diff
@@ -21,18 +21,22 @@ const GroupChatChannelItem = (props: ChannelItemProps) => {
   const hasBadge = unreadCount > 0;
 
   const description = groupChat?.rawJson?.firstMessage ?? groupChat?.rawJson?.message;
-  const isSystemDescription = description?.isSystem ?? description?.type === 'system';
+  const isSystemDescription =
+    description?.isSystem ||
+    description?.type === 'system' ||
+    description?.message?.type === 'system';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for determining if a message is a system message has been expanded. This change seems to be more comprehensive and should cover more cases where a message could be considered a system message.
```diff
- const isSystemDescription = description?.isSystem ?? description?.type === 'system';
+ const isSystemDescription =
+   description?.isSystem ||
+   description?.type === 'system' ||
+   description?.message?.type === 'system';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->